Makefile

```diff
@@ -12,14 +12,14 @@ ${PROG}:	${PROG}.c
 	@OSTYPE=`uname`; \
 	OS_CFLAGS=-D__OS_$${OSTYPE}__; \
 	if [ $$OSTYPE = "FreeBSD" ]; then \
-		OS_CFLAGS="$$OS_CFLAGS -DHAS_LCHMOD -DHAS_CHFLAGS -DHAS_FCHFLAGS -DHAS_LCHFLAGS -DHAS_POSIX_FALLOCATE ${HAS_FREEBSD_ACL}"; \
+		OS_CFLAGS="$$OS_CFLAGS -DHAS_LCHMOD -DHAS_CHFLAGS -DHAS_FCHFLAGS -DHAS_LCHFLAGS -DHAS_POSIX_FALLOCATE -DHAS_UTIMENSAT ${HAS_FREEBSD_ACL}"; \
```

Comment:
I understand the concern if this code was going to remain in the repo as-is, but I have some work queued up in the next few days that will convert all of this magic hardcoded "HAS_X" goop to autoconf-defined preprocessor macros, which will fix this as well in a more holistic way.

I'm waiting for the Travis CI part to be done, and this part, because I'm basing multiple branches on each other (which makes it hard to squash commit/reword commits).